pkg/artifactory/provider/provider.go

```diff
@@ -0,0 +1,4 @@
+package provider
+
+var Version = "9.0.0" // needs to be exported so make file can update this
+var productId = "terraform-provider-artifactory/" + Version
```

Comment:
Should this be capitalized?

Reply:
no, `productId` is only used within the `provider` package.